test/acceptance/behave/components/ngsiv2/entities/update_existing_entity_attributes/update_existing_entity_attributes_part3.feature

```diff
@@ -445,15 +445,14 @@ Feature: update an attribute by entity ID if it exists using NGSI v2. "PATCH" -
     And properties to entities
       | parameter       | value       |
       | attributes_name | temperature |
-    #  | attributes_value | 45          |
       | metadatas_name  | my_meta     |
       | metadatas_value | 5678        |
     When update attributes by ID "room" if it exists and with "normalized" mode
     Then verify that receive an "No Content" http code
     And verify that an entity is updated in mongo
 
-  @attribute_metadata_replace @BUG_1788 @skip
-  Scenario:  update an attribute by entity ID using NGSI v2 and append a new metadata without metadata type
+  @attribute_metadata_replace_2 @BUG_1788
```

Comment:
`@attribute_metadata_replace_2` is not a very good name...

Maybe this should be renamed to `@attribute_metadata_replace_with_md_type` and `@attribute_metadata_replace` to `@attribute_metadata_replace_without_md_type` ?